Makefile

```diff
@@ -26,7 +26,7 @@ MAIN_PACKAGE ?= ./cmd/controller/
 
 COMMIT_HASH ?= $(shell git rev-parse --short HEAD 2>/dev/null)
 BUILD_DATE ?= $(shell date +%FT%T%z)
-VERSION ?= 0.3.11
+VERSION ?= 0.3.12
```

Comment:
This should really come from the git tag IMO.

Created #66 to fix that, so this is not a blocker.

Reply:
Yeah this needs to be fixed, thanks for creating the issue. We can come back to fix this later.